spec/components/courses/financial_support/scholarship_and_bursary_component_spec.rb

```diff
@@ -38,6 +38,48 @@
         expect(result.text).not_to include('With a scholarship or bursary, you’ll also get early career payments of £2,000')
       end
     end
+
+    context 'when course has a scholarship' do
+      let(:course) {
+        build(:course,
+              subjects: [
+                build(:subject,
+                      subject_name: 'chemistry',
+                      scholarship: 2000,
+                      bursary_amount: 3000,
+                      early_career_payments: 2000),
+              ]).decorate
+      }
+
+      it 'renders link to scholarship body' do
+        result = render_inline(described_class.new(course))
+
+        expect(result.text).to include('For a scholarship, you’ll need to apply through the Royal Society of Chemistry')
+        expect(result).to have_selector(
+          "a[href='https://www.rsc.org/prizes-funding/funding/teacher-training-scholarships/']",
+          text: 'Check whether you’re eligible for a scholarship and find out how to apply',
+        )
+      end
```

Comment:
Suggestion - I only just learnt about the [view component rspec helpers](https://viewcomponent.org/guide/testing.html#rspec-configuration):

```suggestion
        render_inline(described_class.new(course))

        expect(rendered_component).to have_text('For a scholarship, you’ll need to apply through the Royal Society of Chemistry')
        expect(rendered_component).to have_link('Check whether you’re eligible for a scholarship and find out how to apply', href: 'https://www.rsc.org/prizes-funding/funding/teacher-training-scholarships/')
      end
```

Reply:
thanks much nicer, thank you!